keel-sql/src/main/resources/db/changelog/20190913-add-status-to-artifact-version.yml

```diff
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: status-to-artifact-version
+      author: emjburns
+      changes:
+        - addColumn:
+            tableName: delivery_artifact_version
+            columns:
+              - column:
+                  name: status
+                  type: varchar(255)
+                  constraints:
+                    nullable: true
+      rollback:
+        - dropColumn:
+            tableName: delivery_artifact_version
+            columnName: status
+
+  - changeSet:
+      id: status-to-artifact-version-drop-index
+      author: emjburns
+      changes:
+        - dropIndex:
+            indexName: delivery_artifact_version_uid_idx
+            tableName: delivery_artifact_version
+      rollback:
+        - createIndex:
+            indexName: delivery_artifact_version_uid_idx
+            tableName: delivery_artifact_version
+            columns:
+              - column:
+                  name: delivery_artifact_uid
+  - changeSet:
+      id: status-to-artifact-version-create-better-index
```

Comment:
Isn't the version still unique and the status is just supplementary?

Reply:
ya, but @asher advised creating the index this way because it can be used for both types of queries.